web/src/pages/Settings/Streams/StreamModal/StreamModal.jsx

```diff
@@ -6,20 +6,21 @@ import Alert from "@mui/material/Alert";
 import "./StreamModal.scss";
 import StreamTemplates from "../StreamTemplates.json";
 import ModalCard from "@/components/ModalCard/ModalCard";
+import { none } from "ramda";
 
 const NAME_DESC =
   "This name can be anything - it will be used to select this stream from the source selection dropdown";
 const DISABLED_DESC = "Don't show this stream in the input dropdown";
 
 // We're already using mui, why are we reinventing the wheel? https://mui.com/material-ui/react-text-field/
 // if it's a matter of className control on the underlying components, that still works with the mui textfield with the InputLabelProps prop and other componentProps
```

Comment:
https://mui.com/material-ui/react-text-field/
I forgot about us recreating the TextField manually
I still hate it, but this time around I'm gonna make an issue out of that and solve it myself